chef/data_bags/crowbar/template-network.json

```diff
@@ -68,6 +68,14 @@
       ],
       "conduit_map": [
         {
+          "pattern" : "team/4/ceph.*",
+          "conduit_list" : {
+            "intf2" : { "if_list" : [ "?1g1", "?1g3" ] },
+            "intf1" : { "if_list" : [ "?1g2", "?1g4" ] },
+            "intf0" : { "if_list" : [ "?1g1", "?1g3" ] }
```

Comment:
So it reads in a sensible order? :-) Sure, will do (pending any further comments from anyone else to save a rebase).